examples/complex_module/README.md

```diff
@@ -103,7 +103,7 @@ An example configuration for an Arm component, a Gizmo component, and a Summatio
       "name": "mysum1",
       "type": "summation",
       "namespace": "acme",
-      "model": "viam:sum:mysum",
+      "model": "acme:demo:mysum",
```

Comment:
I changed this back to acme because I did not see a `sum` or `summation` as an example of a Viam namespace supported component as per the RDK repo